Views/layout.erb

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Cars!</title>
+    <link rel="stylesheet" href="../styles.css" media="screen" title="no title">
```

Comment:
I think the path should just be relative `/styles.css` since its directly in `Public` and by convention `public` is ignored from the path